packages/augur-core/source/contracts/reporting/Market.sol

```diff
@@ -64,12 +64,13 @@ contract Market is Initializable, Ownable, IMarket {
     function initialize(IAugur _augur, IUniverse _universe, uint256 _endTime, uint256 _feePerCashInAttoCash, uint256 _affiliateFeeDivisor, address _designatedReporterAddress, address _creator, uint256 _numOutcomes, uint256 _numTicks) public beforeInitialized {
         endInitialization();
         augur = _augur;
-        require(msg.sender == augur.lookup("MarketFactory"), "Market: MarketFactory only function called by non-MarketFactory");
+        require(msg.sender == augur.lookup("MarketFactory"));
         _numOutcomes += 1; // The INVALID outcome is always first
         universe = _universe;
         cash = ICash(augur.lookup("Cash"));
         owner = _creator;
         repBondOwner = owner;
+        cash.approve(address(augur), MAX_APPROVAL_AMOUNT);
```

Comment:
Is this to approve the market to move cash?

Reply:
To handle orders as well. Since markets no longer store the funds directly (Universes do)